### PR TITLE
Silverstripe 4 namespace fixes

### PR DESCRIPTION
**Pull request by adrexia**
The only major change to get this module to work was namespaces. I've tested all the scenarios I could think off, and they all worked perfectly. Getting the CMS upload stuff to work was tricky, but that's much less about this module and more about the FileSystem changes. :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,6 +11,6 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "~3.1"
+        "silverstripe/framework": "~4.0.0-alpha7"
```
Comment (stevie-mayhew) [inline composer.json]: @adrexia Can we make this `~4.0` so we dont have to change it later?

**Comment by adrexia** [comment]
Hey @stevie-mayhew, have you had a chance to test this at all? Any changes I need to make?

File: composer.json [hunk 2]
```diff
@@ -11,6 +11,6 @@
         }
     ],
     "require": {
-        "silverstripe/framework": "~3.1"
+        "silverstripe/framework": "~4.0.0-beta1"
```
Comment (stevie-mayhew) [inline composer.json]: Can we make this ^4.0 and I can merge :)

**Review by stevie-mayhew — CHANGES_REQUESTED** [review]

**Comment by adrexia** [comment]
Done!

**Review by stevie-mayhew — APPROVED** [review]

**Comment by adrexia** [comment]
Thanks!

**Pull request merged**